packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -1356,7 +1356,8 @@ export async function saveFile(doc: vscode.TextDocument, datasetProvider: api.IZ
     const profile = Profiles.getInstance().loadNamedProfile(sesName);
     if (!profile) {
         globals.LOG.error(localize("saveFile.log.error.session", "Couldn't locate session when saving data set!"));
-        return api.Gui.errorMessage(localize("saveFile.log.error.session", "Couldn't locate session when saving data set!"));
+        await api.Gui.errorMessage(localize("saveFile.log.error.session", "Couldn't locate session when saving data set!"));
+        return;
```

Comment:
curious why not `return await api.Gui.errorMessage`?
Also... should we type the return value of this (`saveFile`) function to `: Promise<void>`?

Reply:
Currently we are marking `return await` as a warning in ESLint, not sure if we would want to change this as part of #2068 or leave it as-is.

Reply:
Oh, that's great that we have that as a rule.
I'm a fan of one-liners. However, in this particular case (with `return await`) I have gotten confused sometimes and question myself on whether or not I meant to await it or just return the promise and let the caller deal with it.

I'm ok with leaving it as-is 😋 
Thanks for entertaining the conversation 🥳